root/utility/tableColumns.js

```diff
@@ -503,6 +503,49 @@ export const isrcsColumn:
     accessor: 'isrcs',
   };
 
+export const acoustidColumn:
+  ColumnOptions<{+gid?: string, ...}, string> = {
+    Cell: ({cell: {value}}) => {
+      const url = '//api.acoustid.org/v2/track/list_by_mbid' +
+        `?format=json&disabled=1&mbid=${value}`;
+      const [tracks, setTracks] = React.useState([]);
+      const [isLoaded, setIsLoaded] = React.useState(false);
+      React.useEffect(() => {
+        fetch(url).then(
+          resp => resp.json(),
+        ).then(data => {
+          setTracks(data.tracks);
+          setIsLoaded(true);
+        });
+      }, [url]);
```

Comment:
This should work now because it's run in the context of the table component and the column is always there, but I'm afraid it'll violate the [Rules of Hooks](https://reactjs.org/docs/hooks-rules.html) if we make columns hideable at some point or if react-table changes how they render stuff.

We should be able to easily fix this though, just by moving all the `Cell` contents to an `AcoustidCell` component defined above the column here. Then just return `<AcoustidCell value={value} />` from `Cell` here.

Reply:
Done. Please check the types I used, tests pass but I'm not sure I used the right way